renderdoc/replay/replay_driver.h

```diff
@@ -37,6 +37,26 @@ struct FetchFrameRecord
   rdctype::array<FetchDrawcall> drawcallList;
 };
 
+struct GetTextureDataParams
+{
+  bool forDiskSave;
+  FormatComponentType typeHint;
+  bool resolve;
+  bool forceRGBA8unorm;
```

Comment:
We shouldn't have a `REMAP_R8` - any single channel 8-bit format can just be remapped to RGBA8, and we can leave the other channels with default values (filled out with 0,0,0,1 as appropriate).

The idea of this functionality is to remap to as few formats as we can reasonably afford, without losing precision. Technically we could map all colour formats to RGBA32 but including RGBA8 and RGBA16 formats is a minor concession for the sake of less memory/bandwidth overhead and better performance. I don't think we should have every possible number of channels for each bit-depth though.